stream-video-android-core/src/main/kotlin/io/getstream/video/android/core/StreamVideoBuilder.kt

```diff
@@ -113,6 +113,10 @@ public class StreamVideoBuilder @JvmOverloads constructor(
     private val sounds: Sounds = defaultResourcesRingingConfig(context).toSounds(),
     private val crashOnMissingPermission: Boolean = false,
     private val permissionCheck: StreamPermissionCheck = DefaultStreamPermissionCheck(),
+    @Deprecated(
```

Comment:
This will not add a visual indicator (a strikethrough) when using the constructor parameter. But it will show a message in the definition popup:

![image](https://github.com/user-attachments/assets/5e79e304-8efd-47ea-b962-93f94da0252e)

Reply:
Just fyi.

Anyway, the parameter is ignored and the ones from `callServiceConfig` is used.